content/docs/contributing/release-process.md

```diff
@@ -206,7 +209,7 @@ page if a step is missing or if it is outdated.
     3. Create the tag for the new release locally and push it upstream:
 
        ```bash
-       git tag -s -m"v1.8.0-beta.0" v1.8.0-beta.0
+       git tag -m"v1.8.0-beta.0" v1.8.0-beta.0
```

Comment:
note: Having `-s` here was a mixture of an oversight and optimism from me.

Oversight: I assumed we'd all have keys set up just for fun.
Optimism: I figured in the future we'd have some way of signing tags using a specific cert-manager key (perhaps using `cmrel`).

I think it's reasonable to remove the requirement to sign tags here, given we sign our actual releases using a cert-manager key which we publish and all signed tags so far have been done with personal developer keys.